src/main/java/seedu/duke/classes/Category.java

```diff
@@ -0,0 +1,17 @@
+package seedu.duke.classes;
+
+public class Category {
```

Comment:
1. Maybe can use an enum for this? Similar to recurrence. 
![image](https://github.com/nus-cs2113-AY2324S1/tp/assets/93811746/fedff346-92e0-4cb2-81fa-e7ed9b8763b9)